python/lsst/daf/butler/direct_butler/_direct_butler.py

```diff
@@ -2066,7 +2085,7 @@ def validateConfiguration(
         # Find all the registered instruments (if "instrument" is in the
         # universe).
         if "instrument" in self.dimensions:
-            instruments = {record.name for record in self._registry.queryDimensionRecords("instrument")}
+            instruments = set(rec.name for rec in self.query_dimension_records("instrument", explain=False))
```

Comment:
Why `set()` instead of `{}`?

Reply:
because it was `set(records)` to begin with and then I realized I wanted the names...

Reply:
Well, I was asking about change from `{...}` to `set(...)`, they do the same thing?

Reply:
Yes. I know. Looking at the diff it's obvious that I should go back to how it was before to minimize unnecessary churn.